modules/ai-authenticating-against-ai-rest-api.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: PROCEDURE
+[id="ai-authenticating-against-ai-rest-api_{context}"]
+= Authenticating against the Assisted Installer REST API
+
+Before you can use the link:https://console.redhat.com/openshift/assisted-installer/clusters[console.redhat.com] Assisted Installer REST API, you need to authenticate against the API using a JSON web token (JWT) that you generate.
+
+.Prerequisites
+
+* Log in to link:https://console.redhat.com/openshift/assisted-installer/clusters[console.redhat.com] as a user with cluster creation privileges.
+
+* Install `jq`.
```

Comment:
We're dealing with a REST API (not oc) and jq is absolutely a reasonable requirement for a sys admin/developer in this context.

Reply:
This was a request from support because jq is not available on RHEL 7.

Reply:
Ok interesting. In general I follow this guidance in my docs. This was an exception. I'm going to look a bit further into this. Thanks.

Reply:
OK. I verified that `jq` _is_ at least available by default on RHEL 8. However, I do want to proceed with this "install `jq`" directive as is. In this instance, it is "truly required". Also, this is a very high priority epic for Telco in 4.11. I've already peeled off an unsupported on-prem procedure to a KCS article for this epic, I really want to keep this one in place in core docs. Let me know your thoughts @kalexand-rh thanks.

Reply:
If it's truly required, that is fine! It's a good thing to check in on - sometimes the commands can be reworked to avoid that package. I'm glad that they've made the package available in RHEL 8 and that someday this won't be an issue anymore.